src/main/kotlin/lotto/domain/RandomLottoGenerator.kt

```diff
@@ -1,24 +1,20 @@
 package lotto.domain
 
-import java.util.*
-
+import java.util.Random
 
 class RandomLottoGenerator : LottoGenerator {
 
     override fun generate(): Lotto {
-        val lottoBalls = List(Lotto.LOTTO_SIZE) { generateLottoBall() }
-        return Lotto(lottoBalls)
-    }
-
-    private fun generateLottoBall(): LottoBall {
-        val number = RANDOM.nextInt(49)
-        if (number == 0) {
-            return LottoBall(1)
+        val lottoBalls = mutableSetOf<LottoBall>()
+        while (!Lotto.isLottoSize(lottoBalls.size)) {
+            lottoBalls.add(RANDOM.lottoBall(LottoBall.LOTTOBALL_MIN, LottoBall.LOTTOBALL_MAX))
```

Comment:
```suggestion
            lottoBalls.add(RANDOM.lottoBall())
```

lottoBall만 사용할 수 있지 않을까요?

Reply:
List에서 셔플해서 사용하는 방식으로 변경하였습니다..!